cdx_writer.py

```diff
@@ -555,6 +553,14 @@ def aif_meta_tags(self):
 
         return ''.join(s) if s else None
 
+class ResourceHandler(ResponseHandler):
```

Comment:
`ResourceHandler` should not extend `ResponseHandler` (it is for regular HTTP response records, `response` and `revisits`).